server/graphql/v1/mutations/collectives.js

```diff
@@ -98,7 +101,7 @@ export async function createCollective(_, args, req) {
   }
 
   try {
-    collective = await models.Collective.create(omit(collectiveData, ['HostCollectiveId', 'hostFeePercent']));
```

Comment:
Why were we bothering to set `hostFeePercent` and omit it on creation?

Reply:
Exactly, it wasn't being used. As I can see through git blame, that this was omitted in a refactor allowing hosts to change `hostFeePercent`.